sklearn/tree/_export.py

```diff
@@ -714,6 +718,33 @@ def recurse(self, node, tree, ax, max_x, max_y, depth=0):
                     (max_y - node.parent.y - 0.5) / max_y,
                 )
                 ax.annotate(node.tree.label, xy_parent, xy, **kwargs)
+
+                # Draw True/False labels if parent is root node
+                if node.parent.parent is None:
+                    # Adjust the position for the text to be slightly above the arrow
+                    text_pos = (
+                        (xy_parent[0] + xy[0]) / 2 + 0.05,
+                        (xy_parent[1] + xy[1]) / 2,
+                    )
+                    # Annotate with your desired text
+                    if node.parent.left() == node:
+                        ax.annotate(
+                            "True",
+                            text_pos,
+                            ha="center",
+                            va="center",
+                            fontsize=10,
```

Comment:
Does this have to be `self.fontsize` if it is not `None`?

How does this annotation scale when the `figsize` is large, (i.e. `figsize=(20,20)`)

Reply:
I did a refactoring of the kwargs, so that way it shouldn't be hardcoded here. I included in the PR description the image from various figure sizes.